src/flaskr/app/modules/api/admin/put.py

```diff
@@ -29,21 +35,10 @@
 # Method        #
 #################
 
-
 def Put(args):
-    current_password = args['current_password']
-    first_name = args['first_name']
-    last_name = args['last_name']
-    new_password = '' if not args['new_password'] else args['new_password']
+    am = AdminManagement()
+    response = jsonify({"": ""})

```

Comment:
This is unnecessary, let am.create_password make it, if it fails, make a meaningful error message, don't just send blanks.